code/_onclick/ai_onclick.dm

```diff
@@ -204,10 +204,16 @@
 	open_close(user)
 
 /obj/machinery/door/airlock/AICtrlClick(mob/living/silicon/user) // Bolts doors
+
 	if(!ai_control_check(user))
 		return
+	if(!user.mind.special_role)
+		if(do_after(user, 2.5 SECONDS, target = src, allow_moving = TRUE))
+			toggle_bolt(user)
+		return
 	toggle_bolt(user)
```

Comment:
Have you tried this? I think this would make the code cleaner, and I think would technically work the same given how byond handles boolean operators
```suggestion
	if(user.mind.special_role || do_after(user, 2.5 SECONDS, target = src, allow_moving = TRUE))
		toggle_bolt(user)
```

Reply:
Actually, it might be cleaner to just make this a callback, if you want them to allow moving/switching between hands and stuff